packages/config/index.js

```diff
@@ -22,21 +22,24 @@ const getAppEntry = (rootFolder, isProd) => {
 };
 
 module.exports = (configurations) => {
+    const isProd = configurations.isProd || process.env.NODE_ENV === 'production';
     const { insights } = require(`${configurations.rootFolder}/package.json`);
     const gitBranch = process.env.TRAVIS_BRANCH || process.env.BRANCH || gitRevisionPlugin.branch();
-    const appDeployment = configurations.deployment || ((process.env.NODE_ENV === 'production' && betaBranhces.includes(gitBranch)) ?
+    const appDeployment = configurations.deployment || ((isProd && betaBranches.includes(gitBranch)) ?
         'beta/apps' :
         'apps');
 
     const publicPath = `/${appDeployment}/${insights.appname}/`;
-    const appEntry = getAppEntry(configurations.rootFolder, process.env.NODE_ENV === 'production');
+    const appEntry = configurations.appEntry || getAppEntry(configurations.rootFolder, isProd);
```

Comment:
I'm aiming for a single entrypoint for frontend-starter-app in the future. In the meantime allowing the user to pass their own entry point is good. Not sure why this was always locked down to `entry.js` and `entry-dev.js`?